test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -122,72 +117,39 @@ public Task Localization_CustomCulture()
                 "Hello Hello Hello Hello");
         }
 
-        [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Windows)]
-        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        public Task Localization_ResourcesInFolder_ReturnLocalizedValue_Clr()
-        {
-            var testRunner = new TestRunner(_applicationPath);
-            return testRunner.RunTestAndVerifyResponse(
-                RuntimeFlavor.Clr,
-                RuntimeArchitecture.x64,
-                "http://localhost:5072",
-                "ResourcesInFolder",
-                "fr-FR",
-                "Bonjour from StartupResourcesInFolder Bonjour from Test in resources folder Bonjour from Customer in resources folder Bonjour from Test in resources folder");
-        }
-
-        [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Windows)]
-        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        public Task Localization_ResourcesInFolder_ReturnLocalizedValue_WithCultureFallback_Clr()
```

Comment:
This test was removed entirely because we already have a Windows and an AllOS version.